### PR TITLE
git clone

### PR DESCRIPTION
**Pull request by koudaiii**
## REF

https://github.com/wantedly/infrastructure/issues/301
### JSONScheme

https://github.com/wantedly/risu/blob/master/docs/api-v1-alpha.md
### スニペット

https://github.com/wantedly/infrastructure/issues/301#issuecomment-130198828
## WHY

docker build させるため
## WHAT

buildsから受け取ったsource_repoとsource_revisionからgit cloneする

### CODE REVIEW

**Comment by koudaiii** [comment]
## git clone
- Only Public Repos
- Only Latest master branch

File: risu.go [hunk 1]
```diff
@@ -77,6 +91,31 @@ func show(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	ren.JSON(w, http.StatusOK, build)
 }
 
+// Clone run "git clone <repository_URL>" and "git checkout rebision"
```
Comment (spesnova) [inline risu.go]: `rebision` -> `revision`

File: risu.go [hunk 2]
```diff
@@ -2,19 +2,27 @@ package main
...
 	"github.com/julienschmidt/httprouter"
+	"github.com/libgit2/git2go"
 	"github.com/unrolled/render"
 
 	"github.com/wantedly/risu/registry"
 	"github.com/wantedly/risu/schema"
 )
 
+const (
+	GitHubHost           = "github.com/"
```
Comment (awakia) [inline risu.go]: GitHubHost = "github.com/" seems so obvious and I think this is not what you want to describe.
Why don't you use `GitRepositoryHost`?

File: risu.go [hunk 3]
```diff
@@ -90,6 +129,9 @@ func setUpServer() *negroni.Negroni {
 }
 
 func main() {
+	if os.Getenv("GITHUB_ACCESS_TOKEN") == "" {
+		os.Exit(1)
```
Comment (awakia) [inline risu.go:157]:
```
log.Fatal("Please provide 'GITHUB_ACCESS_TOKEN' through environment")
```

**Pull request merged**